tests/payroll/test_models.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from django.db.utils import IntegrityError
+
+
+@pytest.mark.django_db(transaction=True)
+def test_null_salary(salary):
```

Comment:
Great work testing each case for your check constraint! How about adding tests to confirm salaries will successfully be created with only base pay and only extra pay?

Reply:
working on it! spending some time with documentation, I can't find anything about testing that something is Nonetype or doesn't exist. Do you know if that's a thing pytest can do?